client/package.json

```diff
@@ -6,8 +6,8 @@
     "@emotion/react": "^11.11.1",
```

Comment:
Noticed a small bug when clicking the three dots at the top of the timeline window

https://github.com/uoftblueprint/tces/assets/58835213/22d595bb-8e64-4e24-8b40-4d97e062683d